py3status/modules/gcal.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming calendar events from Google Calendar.
+
+Configuration parameters:
+    agenda_days: show calendar events for number of days (default 1)
```

Comment:
> We're using command gcalcli agenda [start] [end] so I went with agenda_days

understandable but I'm not sure the user will pick up on it.  So I still think a better name is helpful